tests/test_trace.py

```diff
@@ -861,7 +861,7 @@ def test_event_with_datetime(wrapped_post):
 def test_send_on_error_only_off_with_error(wrapped_post):
     trace = trace_factory.get_or_create_trace()
     trace.token = 'a'
-    trace.runner = mock.MagicMock()
+    trace.runner = RunnerEventMock()
```

Comment:
This is not really related to your PR or I missed something?

Reply:
fixing the random crashes in the tests

Reply:
👑